### PR TITLE
74 today date component

### PR DESCRIPTION
**Pull request by uluisik**

### CODE REVIEW

File: src/pages/TodayPage/Components/NewTodayCard.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          <Spacer />
+          <Flex color='gray'>{tomorrow}</Flex>
+        </Flex>
+        <Flex cursor='pointer' _hover={{ bg: "gray.200" }} p='2'>
+          <NextWeekIcon color='purple' />
+          <Text as='b' ml='2'>
+            Gelecek hafta
+          </Text>
+          <Spacer />
+          <Flex color='gray'>{nextWeek}</Flex>
```
Comment (kod-man) [inline src/pages/TodayPage/Components/NewTodayCard.tsx:90]: It is text so you need to use Text :) Control others. No need to use Flex component for text

File: src/pages/TodayPage/Components/NewTodayCard.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      borderRadius='2%'
+      border='1px solid'
+      borderColor='gray.200'
+      direction='column'
+      ml='3'
+      w='280px'
+      bg='white'
+    >
+      <Flex p='2'>
+        <Input
```
Comment (kod-man) [inline src/pages/TodayPage/Components/NewTodayCard.tsx:46]: I could not understand this part. Why did you add this Input? What place are you using it?

File: src/pages/TodayPage/Components/NewTodayCard.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  const nextWeek = nextWeeks.toLocaleDateString("en-US", options3);
+  const [todays, setTodays] = useState(today);
+  const deleteHandler = (e: any) => {
+    e.preventDefault();
+    setTodays("");
+  };
+
+  return (
+    <Flex
+      borderRadius='2%'
```
Comment (kod-man) [inline src/pages/TodayPage/Components/NewTodayCard.tsx:37]: use Px instead of %

**Review by kod-man — CHANGES_REQUESTED** [review]

File: src/pages/TodayPage/Components/NewTodayCard.tsx (hunk 2 above)
Comment (uluisik) [reply src/pages/TodayPage/Components/NewTodayCard.tsx:46]: In the original of the page, the input and the placaholder are seen when deleted.
